### ISSUE TITLE
Regarding the changed accuracy

### ISSUE THREAD

**Issue by spartian** [sgrvinod/a-PyTorch-Tutorial-to-Text-Classification#7]
Hey Mr Sagar,

I just noticed that you found a new accuracy of 75.1%. Previously it was 74.8%. I tried comparing the changes in new files with previous files but I could not pin point how the accuracy changed by 0.3%. Did the execution of decrease of learning rate made the difference? Can you please point me out the change you think you made to get this accuracy difference?

Thank you

**Comment by sgrvinod** [comment]
Hi @spartian,

I don't remember exactly what I did last time. Now, I just trained for two epochs since it learns very quickly (because of word2vec vectors, size of dataset, simplicity of the task, etc.). The first epoch is at a LR of 1e-3 and the second at 1e-4 respectively. In my experience, training additional epochs didn't really help. Also, I'm not sure how much of the change in accuracy is just because of the random initialization of the network.

**Comment by spartian** [comment]
Based on your clarification I am a bit amazed to know that you achieved the desired accuracy in just 2 epochs....As per my knowledge, a model should be run at least for some number of epochs because improved accuracy generally occurs at later epochs. Just a suggestion, I think you should plot training and testing loss vs no of epochs......See this question for reference .This will better tell you about the no of epochs.

https://stackoverflow.com/questions/52145992/how-do-we-analyse-a-loss-vs-epochs-graph

I am not familiar with early stopping though. On googling, I understood early stopping surfacely. Did you use any equation for early stopping? Is update of Learning rate applied to achieve this?

**Comment by spartian** [comment]
Thank you Sagar for explanation....
